project-base/docker/conf/docker-compose-mac.yml.dist

```diff
@@ -59,11 +59,10 @@ services:
         ports:
             - "35729:35729"
 
-# Template for consumer
-#   consumer-name:
-#       <<: *common_consumer_configuration
-#       container_name: shopsys-framework-consumer-name
-#       command: consumer-name
+    product-visibility-recalculate-consumer:
+        <<: *common_consumer_configuration
+        container_name: shopsys-framework-product-visibility-recalculate-consumer
+        command: product_visibility_recalculate
```

Comment:
could you please explain (to me here or later in the docs briefly) how does this work? Where is the command defined? Is it the consumer that is defined in `old_sound_rabbit_mq.yml`? Or is it `docker-php-consumer-entrypoint`? I am a little bit lost here

Reply:
Yes it is name of consumer. So if you add new consumer in the future, you will copy this definition and change name of service, container name and command. The last one will be name of consumer defined in configuration, the other ones are optional.

Reply:
nice, thank you :+1: please include this explanation to the docs :innocent:

Reply:
I will